media/css/mozorg/diversity/2021/diversity-2021.scss

```diff
@@ -0,0 +1,475 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/card';
+@import '~@mozilla-protocol/core/protocol/css/templates/card-layout';
+@import '~@mozilla-protocol/core/protocol/css/components/feature-card';
+@import '~@mozilla-protocol/core/protocol/css/components/hero';
+@import '~@mozilla-protocol/core/protocol/css/components/modal';
+@import '~@mozilla-protocol/core/protocol/css/components/split';
+@import '~@mozilla-protocol/core/protocol/css/components/video';
+
+// * -------------------------------------------------------------------------- */
+// Hero image & banner heading
+
+.c-hero-banner {
+    background: url('/media/img/foundation/annualreport/2019/pixel-mobile-bg.svg') right center no-repeat;
+    background-size: contain;
+    margin-bottom: $spacing-lg;
+    padding-top: $spacing-lg;
+
+    .l-highlight {
+        background: $color-green-40;
+        color: $color-black;
+        display: inline-block;
+        font-weight: bold;
+        line-height: 0.7;
+        padding: 0;
+    }
+
+    .right {
+        align-self: flex-start;
+        margin-left: $spacing-sm;
+    }
+
+    h1 {
+        align-items: flex-start;
+        color: $color-black;
+        display: flex;
+        font-weight: normal;
+        justify-content: center;
+        line-height: 0.9;
+        text-align: center;
+        margin-bottom: 0;
+    }
+
+    .left {
+        position: relative;
+        text-align: right;
+    }
+
+    img {
+        display: block;
+        margin-right: 1px;
+        margin-top: $spacing-sm;
+        max-width: $layout-2xl;
+        position: absolute;
+        right: 0;
+    }
+
+    @media #{$mq-md} {
+        background-image: url('/media/img/foundation/annualreport/2019/pixel-bg.svg');
+        background-position: right center;
+        background-size: auto 100%;
+        padding-top: $spacing-2xl;
+        margin-bottom: $spacing-2xl;
+
+        h1 {
+            @include font-size(80px);
+        }
+
+        img {
+            margin-top: 14px;
+            max-width: $content-xs;
+        }
+    }
+
+    @media #{$mq-lg} {
+        h1 {
+            font-size: 7rem;
```

Comment:
nit: could use the font-size mixin here and it'll output rem with px fallback:
`@include font-size(7rem);`